gnucash/python/init.py

```diff
@@ -3,7 +3,7 @@
 import gnucash._sw_core_utils as _sw_core_utils
 from gnucash._sw_core_utils import gnc_prefs_is_extra_enabled
 from gnucash import *
-from gnucash_core import _
+from gnucash.gnucash_core import _
```

Comment:
I thought after `from gnucash import *` gnucash_core would be imported and `from gnucash_core import _` would work. But that is not the case. It needs to be `from gnucash.gnucash_core import _`

Reply:
Right. `from gnucash import *` imports `gnucash_core._` but doesn't create `gnucash_core` as an unqualified namespace.